src/axom/inlet/ConduitReader.cpp

```diff
@@ -26,47 +26,55 @@ namespace axom
 {
 namespace inlet
 {
-YAMLReader::YAMLReader() { }
-bool YAMLReader::parseFile(const std::string& filePath)
+ConduitReader::ConduitReader(const std::string& protocol) : m_protocol(protocol)
+{
+  SLIC_ERROR_IF(
+    (protocol != "yaml") && (protocol != "json"),
+    "Inlet: Only JSON and YAML protocols are supported by ConduitReader");
```

Comment:
The mixed casing between the check and the error is concerning.  Can you lower case it before the check? (using the previously mentioned new string utilities function in some other PR I asked for :) )

Reply:
Or is this hidden behind the yaml/json reader classes from the users?